packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -15,7 +15,9 @@ import {
   metaAttribute,
   MetaConstants,
   useBreakpoint,
+  getStyledProps,
```

Comment:
haha lol I also started with `style-props` (check out the branch name 😆) but because we are so used to things like `styled-components` and `styled-system`, I kept mis-spelling things everywhere. I think it will cause confusion. Also this shouldn't matter for consumer as we're not exposing any of the styled props utilities to consumers. Its limited to what we write in docs.